bnpackage/relente.py

```diff
@@ -0,0 +1,69 @@
+class Cat:
+    def __init__(self, name, breed, food):
+        self.name = name
+        self.breed = breed
+        self.food = food
+        self.energy = 100  # Add an energy attribute
+
+    def choice_1(self):
```

Comment:
Lines 8, 11,14, 21, 25 should have [verb form](https://docs.google.com/presentation/d/1CAruo7EEGJuMzRJp0dLdE-HARqiRSALEfM-cc4grjfU/edit#slide=id.g31a82684eca_0_0).